code/modules/kitchen/tools/tools.dm

```diff
@@ -115,14 +118,14 @@
 
 /obj/item/weapon/material/kitchen/utensil/knife/attack(target as mob, mob/living/user as mob)
 	if ((CLUMSY in user.mutations) && prob(50))
-		user << "\red You accidentally cut yourself with the [src]."
+		user << "<span class='warning'>You accidentally cut yourself with \the [src].</span>"
 		user.take_organ_damage(20)
 		return
 	return ..()
 
 /obj/item/weapon/material/kitchen/utensil/knife/attack(target as mob, mob/living/user as mob)
 	if ((CLUMSY in user.mutations) && prob(50))
-		user << "\red You somehow managed to cut yourself with the [src]."
+		user << "<span class='warning'>You somehow managed to cut yourself with \the [src].</span>"
 		user.take_organ_damage(20)
 		return
 	return ..()
```

Comment:
Just tried it and yes, they call each other. So it effectively gives you a 75% chance of cutting yourself if clumsy.

Also: "You somehow managed to cut yourself with **the the** steel knife."